apps/studio/components/interfaces/Settings/Logs/Warehouse.utils.ts

```diff
@@ -0,0 +1,18 @@
+export type WarehouseQueryTemplate = {
+  query: string
+  name: string
+  description: string
+}
+
+export function createWarehouseQueryTemplates(
+  collections: { name: string }[]
+): WarehouseQueryTemplate[] {
+  return collections.map((collection) => ({
+    query:
```

Comment:
reference to https://github.com/supabase/supabase/pull/27611/files#r1666784496

not sure if BE or FE issue but this query doesnt seem to be working: "Cant find source {source name}"